Frontend/ui-common/src/Components/RideCompletedCard/Controller.purs

```diff
@@ -142,7 +142,7 @@ config = {
   theme : DARK,
   isPrimaryButtonSticky : false,
   bannerConfig : Banner.config,
-  showBannerBelowQR : true
+  showBannerAboveQR : false
```

Comment:
@prakharritik instead of maintaining a boolean variable to toggle, We can use a field to decide the order.